src/types/PaymentSheet.ts

```diff
@@ -40,7 +44,20 @@ export declare namespace PaymentSheet {
         merchantCountryCode?: string;
         testEnv?: boolean;
       };
-
+  export interface Address {
+    city?: string;
+    country?: string;
+    line1?: string;
+    line2?: string;
+    postalCode?: string;
+    state?: string;
+  }
+  export interface BillingDetails {
+    address: Address;
```

Comment:
I think I remember Android doing optionality differently here.  Will this interface work for Android, too?  @brnunes-stripe

Reply:
I don't have a snapshot build of Android to really confirm whether or not this will work, so I can't say yet. @brnunes-stripe let me know if you think this will be compatible.

Reply:
Only difference is that `address` is also optional. Can we make it optional on iOS too so the APIs match?
https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt#L129-L144